doc/classes/AudioEffectCapture.xml

```diff
@@ -32,6 +32,7 @@
 			<description>
 				Gets the next [param frames] audio samples from the internal ring buffer.
 				Returns a [PackedVector2Array] containing exactly [param frames] audio samples if available, or an empty [PackedVector2Array] if insufficient data was available.
+				The samples are signed floating point PCM in the range [code][-1, 1][/code]. You will have to scale them if you want to use them as 8 or 16-bit integer samples. ([code]v = 0x7fff * samples[0].x[/code])
```

Comment:
To be honest I have no clue what PCM (Pulse-code modulation) wants to mean here, but expressing the range that way is a bit unusual.
```suggestion
				The samples are signed floating-point PCM between [code]-1[/code] and [code]1[/code]. You will have to scale them if you want to use them as 8 or 16-bit integer samples. ([code]v = 0x7fff * samples[0].x[/code])
```

Reply:
It's a standard math notation for intervals, we use it in several places in the docs, e.g.:
```
doc/classes/FileAccess.xml
374:                            [b]Note:[/b] The [param value] should lie in the interval [code][0, 255][/code]. Any other value will overflow and wrap around.
383:                            [b]Note:[/b] The [param value] should lie in the interval [code][0, 2^16 - 1][/code]. Any other value will overflow and wrap around.
424:                            [b]Note:[/b] The [param value] should lie in the interval [code][0, 2^32 - 1][/code]. Any other value will overflow and wrap around.
```

But spelling it out as suggested isn't bad either.

Reply:
You could expand `PCM (Pulse-code modulation)`.